package.json

```diff
@@ -72,8 +72,7 @@
     "pre-commit": "lint-staged"
   },
   "lint-staged": {
-    "*.js": "trunk check --fix --filter=eslint",
-    "*.md": "trunk check --fix --filter=markdownlint",
+    "*": "trunk check --fix",
```

Comment:
This means every single file type will be run through Trunk, and I'm not sure we want that at this point. 

Let's go slow and just stick with JS and Markdown. We can explore other file types later.